### PR TITLE
Formatting And Grammar

### PR DESCRIPTION
**Pull request by death7654**
- Fixes Linting Errors
- '*' to '-'
- Contractions are removed and replaced with their extended forms

I apologize in advance for all the Email Notifs I may have caused

### CODE REVIEW

**Comment by ninelore** [comment]
Please do not edit markdownlint rules. They are like that for a reason.

Lighthouse check pending

**Comment by death7654** [comment]
I disabled the duplicate name check, and the table check

**Comment by ninelore** [comment]
Yes and both have Implications that we don't want. Please fix the Issue, not the Check

File: src/docs/exiting-developer-mode.md [hunk 1]
```diff
@@ -1,30 +1,31 @@
-# Exiting Developer Mode
+# Developer Mode
```
Comment (ethanaobrien) [inline src/docs/exiting-developer-mode.md:1]: This should specify what about developer mode we're going to be talking about. line 28 still titles "Entering", this should be titles "exiting"

File: src/docs/faq.md [hunk 2]
```diff
@@ -7,105 +7,134 @@ Please read this page before asking for help. Your help request will be pointed
...
-* Ubuntu and Ubuntu-based distros may have issues and are not supported, try switching to another distro. Debian is a great alternative if you are used to Ubuntu.
+
+- Ubuntu and Ubuntu-based distros may have issues and are not supported, try switching to another distro. Debian is a great alternative if you are used to Ubuntu.
 
 ### Why are Ubuntu/Ubuntu based distros not supported?
-* They consistently break packages.
-* They have started to force snap on their users.
-* They have published an LTS distro with a non-LTS kernel.
+
+- They consistently break packages.
```
Comment (ethanaobrien) [inline src/docs/faq.md:112]: (Not really a recommendation just an ubuntu hate comment)

That and their operating system live iso is literally as large as windows. Like go use windows at that point, youre getting the same privacy and storage usage.

File: supported-devices/index.js [hunk 3]
```diff
@@ -35,32 +35,35 @@
 
 */
 
-
```
Comment (ethanaobrien) [inline supported-devices/index.js:38]: Why are we changing 4 space tab substitution down to 2. Can we please change it back.

**Review by ethanaobrien — CHANGES_REQUESTED** [review]

File: src/docs/exiting-developer-mode.md (hunk 1 above)
Comment (marcsadler) [reply src/docs/exiting-developer-mode.md:1]: exiting-developer-mode.md is not sourced in the docs. I removed it when I migrated the info in here to the FAQ. I don't know if it's still there after the edits to the FAQ.

**Review by ninelore — COMMENTED** [review]
Fixed pending issues

File: src/docs/exiting-developer-mode.md (hunk 1 above)
Comment (ninelore) [reply src/docs/exiting-developer-mode.md:1]: The page can be found when using the search or by its URL, but its not referenced anywhere else. Feel free to remove it via a new PR

**Review by ninelore — APPROVED** [review]

**Review by ethanaobrien — APPROVED** [review]

File: supported-devices/index.js (hunk 3 above)
Comment (ethanaobrien) [reply supported-devices/index.js:38]: Is there a reason for this?

**Comment by ethanaobrien** [comment]
Me when changes are merged before a review I placed asking a question about the code formatting. At least an answer would be nice.

File: supported-devices/index.js (hunk 3 above)
Comment (death7654) [reply supported-devices/index.js:38]: I think the linter was complaining about it

Comment (ethanaobrien) [reply supported-devices/index.js:38]: Interesting how the linter would be complaining about this. I looked at the ecma standard and it said that the spacing needs to be consistent, and didn't specify 2 or 4 spaces. Doesn't make much sense to change it from the style more commonly used in the documentation

Comment (ninelore) [reply supported-devices/index.js:38]: IDE Formatting sometimes does this

**Pull request merged**